airflow/providers/apache/spark/hooks/spark_jdbc.py

```diff
@@ -31,7 +31,8 @@ class SparkJDBCHook(SparkSubmitHook):
 
     :param spark_app_name: Name of the job (default airflow-spark-jdbc)
     :type spark_app_name: str
-    :param spark_conn_id: Connection id as configured in Airflow administration
+    :param spark_conn_id: The :ref:`spark connection id<howto/connection:spark>`
```

Comment:
```suggestion
    :param spark_conn_id: The :ref:`spark connection id <howto/connection:spark>`
```

Reply:
I'm not sure that a space is needed here?

Reply:
It reads better with a space though, I’d prefer having it added.

Reply:
I can add spaces